pkg/operations/kubernetesupgrade/upgradecluster.go

```diff
@@ -70,7 +70,7 @@ func (uc *UpgradeCluster) UpgradeCluster(subscriptionID uuid.UUID, resourceGroup
 	}
 
 	switch ucs.OrchestratorProfile.OrchestratorVersion {
```

Comment:
@JiangtianLi and I discussed this briefly yesterday. 

My understanding is, I will not touch the logic of upgrade in general. Only in a few places, where I replace version consts with plain strings. The current upgrade feature is not working anyway. So I will leave it later few. 

Another dimension is, upgrade so far is only handling scenario of kubernete 1.5.3 -> 1.6.2. However, 1.5.3 will not be supported, 1.5.7 is the latest of 1.5 minor. And it will never upgrade to 1.6.2, because 1.6.6 is the latest of 1.6 minor.